spec/platform_spec.rb

```diff
@@ -29,8 +29,14 @@ module Pod
       end
 
       it 'can be initialized with a string symbolic name' do
-        Platform.new('ios')
-        @platform.name.should == :ios
+        platform = Platform.new('ios')
```

Comment:
this was initializing a new platform and throwing it away, and then checking state initialized elsewhere. not an effective test